### ISSUE TITLE
Entire application crash when working with frames

### ISSUE THREAD

**Issue by n303ef** [cefsharp/CefSharp.Dom#42]
### Description

When working with frames on some websites, I get entire application crash (it closed completely without any error alerts).
My initial setup to access frames:
```csharp
settings.CefCommandLineArgs.Add("disable-features=IsolateOrigins");
settings.CefCommandLineArgs.Add("disable-site-isolation-trials");
```
It doesn't matter for this issue (crash occurs even without these settings), but they are required to have access to frames.

### Crash moment

Here is code which leads to crash:

```csharp
foreach (Frame fr in devTools.Frames)
{
	if (fr == null || fr.Detached)
		continue;
	ElementHandle element = fr.QuerySelectorAsync("#id").Result;
	if (element != null) //crashes on this line
	{
		return element;
	}
}
```
Strange thing is that it works on most websites, but on some ones it leads to crash every time.

### Logs
In Logs, there are no fatal entries, but only few warning and error ones:
`[0927/180147.326:ERROR:ssl_client_socket_impl.cc(983)] handshake failed; returned -1, SSL error code 1, net_error -101
[0927/180147.399:ERROR:ssl_client_socket_impl.cc(983)] handshake failed; returned -1, SSL error code 1, net_error -101
[0927/180147.554:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 21
[0927/180147.555:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 1
[0927/180147.585:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 25
[0927/180147.797:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 29
[0927/180147.797:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 5
[0927/180147.797:WARNING:spdy_session.cc(3502)] Received HEADERS for invalid stream 31
[0927/180148.081:ERROR:ssl_client_socket_impl.cc(983)] handshake failed; returned -1, SSL error code 1, net_error -101
[0927/180148.157:ERROR:ssl_client_socket_impl.cc(983)] handshake failed; returned -1, SSL error code 1, net_error -101`

**Comment by amaitland** [comment]
> Strange thing is that it works on most websites, but on some ones it leads to crash every time.

You'll need to find a site that reliably reproduces the problem and analyze what the difference in structure for the frames, work out how to create a self contained example.
Then submit a PR that adds an xunit test that reproduces the problem.

> Versions
> Which version of CefSharp.Dom are you using?
> Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.1 or .NET Core 3.1.

Please provide version information as per the bug report temple.

**Comment by n303ef** [comment]
This was not CefSharp.Dom related issue, CefSharp crashes even without any calls after opening website. Sorry for disturbing.

**Comment by n303ef** [comment]
And it also fixes when updating to 105.3.390.
